src/pages/api-documentation.tsx

```diff
@@ -231,7 +232,7 @@ const ApiDocumentation: React.FC = () => {
           </div>
         </DialogTitle>
         <DialogContent dividers>
-          <Typography>Install MetaMask for your platform and refresh the page. The interactive features in this documentation require installing the MetaMask extension. <b onClick={() => setShowInstallDialog(false)} >Proceed without MetaMask</b></Typography>
+          <Typography>Install MetaMask for your platform and refresh the page. The interactive features in this documentation require installing the MetaMask extension. <Link variant="body2" component="button" onClick={() => setShowInstallDialog(false)} >Proceed without MetaMask</Link></Typography>
```

Comment:
Nit: The docs seem to suggest that using a button is preferred if there's no `href`: https://mui.com/components/links/#accessibility

Reply:
I went with `component="buttton"` to hopefully cover that case